pkgs/top-level/all-packages.nix

```diff
@@ -25651,6 +25651,8 @@ with pkgs;
 
   osrm-backend = callPackage ../servers/osrm-backend { };
 
+  overseerr = callPackage ../servers/overseerr { };
```

Comment:
Not a blocker, but consider putting the package in [`pkgs/by-name`](https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name). That's where most new packages go these days.